.travis.yml

```diff
@@ -6,6 +6,7 @@ cache:
   directories:
     - /home/travis/.yarn-cache/
 script:
+  - pip install "jupyterlab<2.0.0"
```

Comment:
Could you please change it to `jupyterlab~=1.0` and update the `extras_require[test]` in the `setup.py` accordingly?

Reply:
`jupyterlab~=1.0` means the same thing as `jupyterlab<2.0.0`, right? I'm fine with making this change

Reply:
Done!